modules/shadertools/src/lib/resolve-modules.js

```diff
@@ -59,14 +59,16 @@ function getDependencyGraph({modules, level, moduleMap, moduleDepth}) {
   }
 }
 
-// registers any supplied modules, resolves any names into modules
-// returns a list of modules
 function instantiateModules(modules, seen) {
   return modules.map(module => {
     if (module instanceof ShaderModule) {
       return module;
     }
 
+    assert(
```

Comment:
Use `log.removed`

Reply:
Shadertools currently doesn't depend on probe.gl. I'd rather not add a dependency for this.